types/porep_proof_partitions.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// PoRepProofPartitions TODO(laser)
 type PoRepProofPartitions uint64
 
 const (
+	// TestPoRepProofPartitions TODO(laser)
```

Comment:
TestPoRepProofPartitions is an opaque value signaling that an unknown number of partitions were used when creating a PoRep proof in test mode.